ddterm/app/application.js

```diff
@@ -659,14 +659,26 @@ class Application extends Gtk.Application {
                 false
             );
 
+            if (!data_variant.is_normal_form()) {
+                logError(new Error('Invalid session file data variant, delete session file.'));
+                GLib.unlink(this.session_file_path);
```

Comment:
If you handle errors using `catch {}` on the top level, you could just `throw` an error here instead

Reply:
changed it to throw an error and try/catch the errors in `restore_session()`, I moved try/catch to  `restore_session()`, in case it is called from somewhere else in the future